src/app/user/registration/_components/owner-form.tsx

```diff
@@ -111,11 +125,16 @@ export default function OwnerForm(props: {
           type="email"
         />
         <BarkFormButton onClick={onRequestOtp}>Send me an OTP</BarkFormButton>
-        {recipientEmail !== "" && (
+        {otpState.status === "SEND_SUCCESS" && (
           <BarkFormParagraph>
-            An OTP has been sent to {recipientEmail}
+            An OTP has been sent to {otpState.email}
           </BarkFormParagraph>
         )}
+        {otpState.status === "SEND_FAILED" && (
+          <BarkFormErrorParagraph>
+            Failed to send OTP. Please request another.
+          </BarkFormErrorParagraph>
+        )}
```

Comment:
Just curious when `SEND_FAILED` would be triggered. In the event that the email service is down I think asking the user to request for another would not help.

Reply:
I'm assuming its when the OTP has expired

Reply:
SEND_FAILED in this case can arise from an exception caught in the email service (Ref1).

You're right that it might not help for a user to keep retrying. However, that is the only option we have at the moment.

That said, we use AWS SES as our SMTP server and so the chance of a prolonged outage should be negligible. Having the user retry should work for most chance network issue.

___

Ref1: where an error may originate from
https://github.com/barkbank/barkbank-pawtal/blob/36bbf03a1fab1b62a9fbc806c1e1df628662a238/src/lib/services/email.ts#L50-L65

Reply:
Thanks! Got it